lib/Runtime/Library/JSON.cpp

```diff
@@ -397,23 +397,22 @@ namespace JSON
         objectStack = Anew(tempAlloc, JSONStack, tempAlloc, scriptContext);
     }
 
-    Js::Var StringifySession::Str(uint32 index, Js::Var holder)
+    Js::Var StringifySession::Str(uint32 index, Js::JavascriptArray * holder)
```

Comment:
Why have you changed the `holder` to be a `JavascriptArray`? From what I recall of the spec, it's potentially an object, but not necessarily an array.

Reply:
See, I think, https://www.ecma-international.org/ecma-262/8.0/index.html#sec-serializejsonproperty

Reply:
Agree with comments. I'm more interested with the optimized path for javascriptArray types only. i.e. better consider proxy separately?

This wasn't planned to be part of 1.7 and no rush.